AAI/AAIConnectProfile.md

```diff
@@ -396,6 +397,10 @@ the Broker.
     presented and/or transformed without misrepresenting the original intent,
     except for accommodating for `exp` timestamps to be represented as
     indicated above.
+    
+3.  An Embedded Token Issuer MAY include the `aud` field to indicate which
+    [Brokers](#term-broker) were consented by the user. The values within the `aud`
+    field must match the `iss` field of OIDC access tokens issued by consented Brokers.
```

Comment:
Based on previous feedback on the spec, we try to avoid the term "OIDC access token" due to the various layers of specs involved. Just "access tokens" here is preferred based on those previous nits and not to undo that previous nit-fixing work.